### ISSUE TITLE
Menu tabs, namely Encounter Summary, Administrative and Clinical dont't work properly in Fees > Fee Sheet menu

### ISSUE THREAD

**Issue by 2208Abhinav** [LibreHealthIO/LibreEHR#1090]
1)Select any past encounter
2)From **Fees** select **Fee Sheet**

In the menu click on **Administrative** or **Clinical** and further click on anything, nothing happens, except when you click on **Encounter Summary**.
![screenshot from 2018-03-30 02-55-13](https://user-images.githubusercontent.com/30211121/38099438-3822dd80-3398-11e8-9745-499499d9f355.png)

But something interesting happens when you go to the same menu(Fee Sheet) after selecting Past Encounter and then following **Administrative** > **Fee Sheet**, every menu tab works fine.

**Comment by teryhill** [comment]
Works fine in mine if I am in the right place.

![image](https://user-images.githubusercontent.com/4655366/38100432-633e7f94-334b-11e8-8724-a41ecefe603d.png)

**Comment by 2208Abhinav** [comment]
Which path did you follow @teryhill ?

**Comment by teryhill** [comment]
I went to fees then fee sheet.

**Comment by 2208Abhinav** [comment]
There is a problem in mine then, should we ask others?

**Comment by teryhill** [comment]
Yes get others to check. My code could be different. I suspect that a bootstrap or fancybox change might have been made in the program in another "Path" and it could be causing issues here. Look at 1087 and you can see what I mean. It is vital to test with the entire system.

**Comment by 2208Abhinav** [comment]
@naveen17797 @apooravc please see this issue.

**Comment by muarachmann** [comment]
Mine works fine

**Comment by apooravc** [comment]
Mine works fine as well.

**Comment by 2208Abhinav** [comment]
Mine os is Ubuntu 64-bit and I am testing on firefox.

**Comment by 2208Abhinav** [comment]
It seems like only I am facing this issue. @teryhill should I close this issue.

**Comment by naveen17797** [comment]
![screenshot from 2018-03-30 11-22-19](https://user-images.githubusercontent.com/18109258/38126224-c810de00-340c-11e8-83b4-803b7c9d6511.png)
ubuntu 64 bit 16.04 on chromium

**Comment by 2208Abhinav** [comment]
@naveen17797 did you click on Administrative and then further on any options like Procedure Order

**Comment by naveen17797** [comment]
Nope

**Comment by 2208Abhinav** [comment]
@naveen17797 do it and tell whether they work or not

**Comment by naveen17797** [comment]
If i click on adminstrative menu it does not work, but if i click on encounter summary and then on adminstrative menu it works

**Comment by 2208Abhinav** [comment]
Yes @naveen17797 thats what I am saying, something weird is happening there.

**Comment by teryhill** [comment]
Based on the video, it appears that there is a resource issue, IE Fancybox , jquery etc. It would great if one or all of y'all look into it. I may not have the most updated code from the last GCI secession.

**Comment by 2208Abhinav** [comment]
@teryhill we will take care of it :smiley:

**Comment by 2208Abhinav** [comment]
@teryhill I will try to solve it

**Comment by aethelwulffe** [comment]
I can't replicate this folks.  Tried every combo.

**Comment by muarachmann** [comment]
@aethelwulffe check this video http://muarachmann.000webhostapp.com

**Comment by aethelwulffe** [comment]
Ah, thanks.  Well, that looks like the stuff.

I didn't (and never do) have the fees enabled in the menu...

I will re-test tomorrow.  Looks like we should be able to figure this 
one (out first JS error probably)
